rust/hyperlane-base/src/db/rocks/hyperlane_db.rs

```diff
@@ -312,3 +313,10 @@ make_store_and_retrieve!(pub, processed_by_nonce, NONCE_PROCESSED, u32, bool);
 make_store_and_retrieve!(pub(self), processed_by_gas_payment_meta, GAS_PAYMENT_META_PROCESSED, InterchainGasPaymentMeta, bool);
 make_store_and_retrieve!(pub(self), interchain_gas_expenditure_data_by_message_id, GAS_EXPENDITURE_FOR_MESSAGE_ID, H256, InterchainGasExpenditureData);
 make_store_and_retrieve!(pub(self), interchain_gas_payment_data_by_message_id, GAS_PAYMENT_FOR_MESSAGE_ID, H256, InterchainGasPaymentData);
+make_store_and_retrieve!(
+    pub,
+    pending_message_data_by_message_id,
```

Comment:
nit: it's not clear that data == retry count unless if you look at how these functions are called, maybe this should be `pending_message_retry_count_by_message_id`?